src/gateway/understanding-kong/key-concepts/plugins.md

```diff
@@ -1,6 +1,9 @@
 ---
-title: plugins
-
+title: Plugins
+concept_type: explanation
 ---
 
-## PLACEHOLDER plugins
+{% include_cached /md/about-plugins.md %}
+
+## See also
```

Comment:
Adding links to the other key-concepts docs here would be a good idea. Maybe a section titled ##Other key concepts ?

Reply:
@Guaris What other key-concept docs were you thinking of? In the include file, the last few sections already contain links to the PDK, plugin development guide, plugin hub contribution guidelines, and plugin other language support, so I wouldn't want to repeat those. I can probably add this Plugin Hub link to the include file so it's in all the "plugin overview" topics.

Reply:
We also have this page: https://docs.konghq.com/konnect-platform/key-concepts/

It's not really well-named; at this point, we could rename it to release lifecycle terms or something similar. It used to have more terms but these are all that's left now.

Reply:
<google-sheets-html-origin><style type="text/css"><!--td {border: 1px solid #ccc;}br {mso-data-placement:same-cell;}--></style>

Key concepts |   |   |  
-- | -- | -- | --
  | Services |   | explanation
  | Routes |   | explanation
  | Upstreams |   | explanation
  | Plugins |   | explanation

The services, routes, and upstreams doc.